packages/react/src/components/Menu/Menu.stories.js

```diff
@@ -45,9 +45,11 @@ export const Playground = (args) => {
   const selectableOnChange = action('onChange (MenuItemSelectable)');
   const radioOnChange = action('onChange (MenuItemRadioGroup)');
 
+  // eslint-disable-next-line ssr-friendly/no-dom-globals-in-react-fc
   const target = document.getElementById('storybook-root');
```

Comment:
Honestly I'd be fine excluding *.stories.js altogether in the plugin config. I don't think there's a reason we need it checking the story files.

Reply:
Same thing with the `*-test.js` files actually, I don't see a reason to run this on them.

Reply:
Yea, will make that update 👍